### PR TITLE
toggle handles ngModel

### PR DESCRIPTION
**Pull request by mikerodonnell89**
#### Please provide a link to the associated issue.

#421 

Add support for ngModel on toggle

### CODE REVIEW

File: library/src/lib/toggle/toggle.component.ts [hunk 1]
```diff
@@ -15,13 +23,33 @@ export class ToggleComponent implements OnInit {
...
-    @Input()
-    id: string;
+    onChange: any = () => {};
+    onTouched: any = () => {};
+
+    get inputText() {
+        return this.checked;
+    }
+
+    set inputText(value) {
```
Comment (MattL75) [inline library/src/lib/toggle/toggle.component.ts]: Maybe a bit confusing to have get and set InputText on a toggle?

**Comment by fkolar** [comment]
I think you dont really need this part

```
if (this.size && this.size !== 'xs' && this.size !== 's' && this.size !== 'l') {
            this.size = null;
        }
```

if you do something like:
```
size: InputSize = 'small';
export type InputSize = 'large' | 'normal' | 'small';
```

**Comment by fkolar** [comment]
But I guess all this should be generic thing common for all comments.

File: library/src/lib/toggle/toggle.component.ts (hunk 1 above)
Comment (mikerodonnell89) [reply library/src/lib/toggle/toggle.component.ts]: fixed

**Pull request merged**